src/main/java/sda/training/rps/app/StageService.java

```diff
@@ -1,49 +1,141 @@
 package sda.training.rps.app;
 
-import sda.training.rps.dao.IStageDao;
 import sda.training.rps.dao.StageDao;
 import sda.training.rps.model.Game;
 import sda.training.rps.model.Stage;
 import sda.training.rps.util.Move;
 import sda.training.rps.util.Result;
 import sda.training.rps.util.ScannerSingleton;
 
-public class StageService implements IStageService {
-    private IStageDao stageDao = new StageDao();
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class StageService {
+    private final StageDao stageDao = new StageDao();
     private Move playerMove;
     private Move computerMove;
-    private Result[][] outcome;
-    private ScannerSingleton scanner = ScannerSingleton.getInstance();
+    public final Result[][] outcome = getOutcome();
+    private final ScannerSingleton scanner = ScannerSingleton.getInstance();
+    private final PlayerService playerService;
+
+    public StageService(PlayerService playerService) {
+        this.playerService = playerService;
+    }
+
+    private Result[][] getOutcome() {
+        return new Result[][]{
+                {Result.DRAW, Result.LOSE, Result.LOSE, Result.LOSE, Result.WIN, Result.WIN, Result.WIN},
```

Comment:
chyba rozumiem pomysl, to jest taka mapa zaleznosci co z czym wygrywa tak ? 
troche malo to czytelne :( 

zaproponuje inny pomysl :) 
Mapa kto z kim wygrywa, jesli czegos tam nie ma to znaczy ze przegrywasz. 
to wydaje sie dosc naturalne map.get(scisors) powinienes dostac liste z czym wygrywaja nozyczki, latwo to przeszukac a i zapis byl by calkiem czytelny 

map.put(scisors, List.of(paper, salami, morkyHerbatnik)) <- od razu z kim wygrywa